build/versionize-docs.sh

```diff
@@ -68,8 +68,26 @@ for doc in "${mdfiles[@]}"; do
   fi
 done
 
-# Update API descriptions to match this version.
-$SED -ri -e "s|(releases.k8s.io)/[^/]+|\1/${NEW_VERSION}|" pkg/api/v[0-9]*/types.go
+# Update kubectl cmd files so that kubectl docs generated from them are as
+# expected.
+go_dirs=(pkg/kubectl/cmd)
+go_files=()
+for dir in "${go_dirs[@]}"; do
+  go_files+=($( find "${dir}" -name "*.go" -type f ))
+done
+# Update API descriptions as well
+go_files+=(pkg/api/v[0-9]*/types.go)
+go_files+=(pkg/api/unversioned/types.go)
+go_files+=(pkg/apis/*/v[0-9]*/types.go)
+go_files+=(pkg/apis/*/types.go)
+
+for file in "${go_files[@]}"; do
+  $SED -ri \
+      -e "s|(releases.k8s.io)/[^/]+|\1/${NEW_VERSION}|g" \
+      -e "s|(${HTML_PREVIEW_PREFIX}/HEAD)|${HTML_PREVIEW_PREFIX}/tree/${NEW_VERSION}|g" \
+      "${file}"
+done
 
 ${KUBE_ROOT}/hack/update-generated-docs.sh
 ${KUBE_ROOT}/hack/update-generated-swagger-docs.sh
+./hack/update-api-reference-docs.sh
```

Comment:
Per offline discussion, ${KUBE_ROOT}/hack/update-api-reference-docs.sh works on my machine, but not on nikhil's. Let's file an issue and solve it later. Thank you.